src/test/ui/consts/partial_qualif.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0492]: cannot borrow a constant which may contain interior mutability, create a static instead
+error[E0492]: this borrow to an interior mutable value may end up in the final value of this constant
```

Comment:
I guess the obvious response one might have to this error message is "so what?". I do not have a proposal for a better wording though, so I am okay with going with this for now.

Reply:
yea I was considering adding another `note: ` to it, or at least use the `--explain` system. I'll do the latter now I guess

Reply:
How do you like the lastest version?